plugin.xml

```diff
@@ -70,7 +70,7 @@ SOFTWARE.
             </intent-filter>
         </config-file>
         <source-file src="src/android/io/branch/BranchSDK.java" target-dir="src/io/branch" />
-        <framework src="io.branch.sdk.android:library:1.+" />
+        <framework src="io.branch.sdk.android:library:1.+" custom="true" type="gradleReference" />
```

Comment:
Hmm..I don't think this would work, these tags are meant for adding .gradle file dependencies ? For any dependency from maven repo we need to add `<framework src="io.branch.sdk.android:library:1.+" />` and it should work.

Reference -> https://cordova.apache.org/docs/en/latest/plugin_ref/spec.html#framework